src/styles/main.scss

```diff
@@ -6,6 +6,7 @@ html {
     font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
     font-size: 62.5%;
     font-weight: 300;
+    line-height: 1.42857143;
```

Comment:
Might be clearer to use SASS's math ability to just make it 20 / 14. But I'm gonna merge.